scripts/data_prep/convert_text_to_mds.py

```diff
@@ -343,14 +340,32 @@ def convert_text_to_mds(
         output_folder (str): Folder to write MDS shards to
         input_folder (str): Folder of text files to process
         concat_tokens (int): Concantenate up to this many tokens
-        eos_text (str): Textend to append to each example to separate concatenated samples
-        bos_text (str): Text to prepend to each example to separate concatenated samples
         no_wrap: (bool): Whether to let text examples wrap across multiple training examples
         compression (str): The compression algorithm to use for MDS writing
         processes (int): The number of processes to use.
         args_str (str): String representation of the arguments
         reprocess (bool): Whether to always reprocess the given folder of text files
+        bos_text (Optional[str]): Text to prepend to each example to separate concatenated samples
+            If None, use the tokenizer's bos_token if tokenizer.add_bos_token is True, otherwise use an empty string.
+        eos_text (Optional[str]): Text end to append to each example to separate concatenated samples
+            If None, use the tokenizer's eos_token.
```

Comment:
don't love that these defaults aren't mirrors of each other, but this allows the defaults to match the finetuning cases we discussed.